bun/inline.go

```diff
@@ -34,25 +37,24 @@ func paintInlineElement(ctx *gg.Context, node *hotdog.NodeDOM) {
 func fetchNodeImage(node *hotdog.NodeDOM) (image.Image, error) {
 	imgPath := node.Attr("src")
 
-	if imgPath != "" {
-		imgURL := sauce.ParseURL(imgPath)
-
-		if imgURL.Scheme == "" && imgURL.Host == "" {
-			imgURL = sauce.ParseURL(node.Document.URL.String() + imgURL.Path)
-		}
-
-		data := sauce.GetImage(imgURL)
-		im, _, err := image.Decode(bytes.NewReader(data))
-
-		if err == nil {
-			return im, nil
-		}
+	imgURL, err := node.Document.URL.Parse(imgPath)
+	if err != nil {
+		return nil, err
+	}
 
-		im, _, _ = image.Decode(bytes.NewReader(assets.ErrorImage()))
-		return im, nil
+	// Fetch the image (either from cache or network)
+	data, err := sauce.GetImage(imgURL)
+	if err != nil {
+		return nil, err
 	}
+	im, _, err := image.Decode(bytes.NewReader(data))
 
-	return nil, errors.New("Failed to fetch " + imgPath)
+	if err != nil {
+		return nil, err
+	}
+	// If we get this far we've failed, so show the standin image.
+	//im, _, _ = image.Decode(bytes.NewReader(assets.ErrorImage()))
```

Comment:
Is this line a leftover commented code?

Reply:
Doh - Yes! I left it there for reference when I was moving it up the call stack and forgot to clean it out. Just force-pushed an update.